templates/install.rdf.mustache

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+  <Description about="urn:mozilla:install-manifest">
+    <em:id>{{addon.id}}</em:id>
+    <em:type>2</em:type>
+    <em:bootstrap>true</em:bootstrap>
+    <em:multiprocessCompatible>{{addon.multiprocessCompatible}}</em:multiprocessCompatible>
+    <em:hasEmbeddedWebExtension>{{addon.hasEmbeddedWebExtension}}</em:hasEmbeddedWebExtension>
+    <em:unpack>false</em:unpack>
+    <em:version>{{addon.version}}</em:version>
+    <em:name>{{addon.name}}</em:name>
+    <em:description>{{addon.description}}</em:description>
+    <em:creator>{{addon.creator}}</em:creator>
+    <em:iconURL>chrome://{{addon.chrome}}/content/icon.png</em:iconURL>
+    <em:icon64URL>chrome://{{addon.chrome}}/content/icon@2x.png</em:icon64URL>
+
+    <!-- Firefox -->
+    <em:targetApplication>
+      <Description>
+        <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
+        <em:minVersion>{{addon.minVersion}}</em:minVersion>
+        <em:maxVersion>{{addon.maxVersion}}</em:maxVersion>
+      </Description>
+    </em:targetApplication>
+
+    <!-- Firefox for Android -->
```

Comment:
I don't think we plan to ship this for Firefox for Android, since Normandy doesn't support it. I wouldn't include this section.